contributors/design-proposals/propagation.md

```diff
@@ -73,13 +79,18 @@ type VolumeMount struct {
 	// Required.
 	MountPath string `json:"mountPath"`
 	// Optional.
-	Propagation string `json:"propagation"`
+	Propagation PropagationMode `json:"propagation,omitempty"`
 }
 ```
 
+Default would be `RSlave`, which should not break backward compatibility,
+`RShared` must be explicitly requested.
+
 Opinion against this:
 
-1. This will affect all volumes, while only HostPath need this.
+1. This will affect all volumes, while only HostPath need this. It could be
```

Comment:
I disagree only hostpath needs propagation. I totally could see it with at least emtpyDir too. Say I want to have a cvmfs sidecar to my main pod? I could use cvmfs with its automounter to mount cvmfs sumbmounts in the emptydir and expose the whole /cvmfs automounter to the other containers in the pod.

I still would really like to see a namedEmptyDir implementation that could use it too.